notebooks/vertex_genai/solutions/evaluation_adk_agent_with_vertex_ai.ipynb

```diff
@@ -0,0 +1,1698 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    BUCKET_NAME = "put_your_ai_agent_evaluation_adk_bucket"
</pre><p>This line will throw an error if a hard-coded bucket name is used since bucket names have to be unique for everyone. My recommendation is to use the PROJECT ID as the bucket name like this instead: </p>

<pre class="ql-syntax" spellcheck="false">PROJECT = !gcloud config list --format 'value(core.project)'
PROJECT = PROJECT[0]
BUCKET_NAME = PROJECT
BUCKET_URI = f"gs://{BUCKET_NAME}"
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>